ui/app/src/components/shared/ResourceCard.tsx

```diff
@@ -72,24 +81,31 @@ export const ResourceCard: React.FunctionComponent<ResourceCardProps> = (props:
     },
   });
 
+  let canConnect = props.resource.properties && props.resource.properties['connection_uri'];
+  let connectUri = canConnect ? props.resource.properties['connection_uri'] : '';
```

Comment:
do we actually need both? 
it's easier to read, for sure, but could we use `connectUri` for all of the tests below?

i.e. do this here...

``` typescript
let connectUri = props.resource.properties && props.resource.properties['connection_uri'];
```